semhash/records.py

```diff
@@ -0,0 +1,60 @@
+from typing import Sequence
+
+from frozendict import frozendict
+
+from semhash.datamodels import DuplicateRecord
+
+
+def to_frozendict(record: dict[str, str], columns: set[str]) -> frozendict[str, str]:
+    """Convert a record to a frozendict."""
+    return frozendict({k: record.get(k, "") for k in columns})
+
+
+def unpack_record(record: dict[str, str], columns: Sequence[str]) -> str:
+    r"""
+    Unpack a record into a single string.
+
+    Uses self.columns to determine the order of the text segments.
+    Each text is cleaned by replacing '\t' with ' '. The texts are then joined by '\t'.
+
+    :param record: A record to unpack.
+    :param columns: Columns to unpack.
+    :return: A single string representation of the record.
+    """
+    parts = []
+    for c in columns:
+        parts.append(record[c].replace("\t", " "))
+    return "\t".join(parts)
```

Comment:
Optimization suggestion since I think a list comprehension is a tiny bit faster usually? Or is this overoptimizing and making the function unreadable? 
```suggestion
    return "\t".join(record.get(c, "").replace("\t", " ") for c in columns)
```